src/Alert/Alert.js

```diff
@@ -1,174 +1,259 @@
-          <div className={classes.footerWrapper}>{props.footer}</div>
+
+        {/* Body Container. */}
+        <div className={classnames('su-order-2 su-flex-1 su-flex-grow', levers.bodyWrapper, classes.bodyWrapper)}>
+
+          {(props.hasIcon && props.isIconTop) && (
+            <span className={classnames('su-mr-5 su-text-left su-ml-0', levers.headerIcon, classes.headerIcon)}>
+              {icon}
+            </span>
+          )}
+
+          {(props.hasLabel && props.isLabelTop) && (
+            <span className={classnames('su-uppercase su-font-bold su-text-17 su-tracking-widest', levers.label, classes.label)}>
+              {props.label ?? 'Information'}
+            </span>
+          )}
+
+          {props.heading && (
+            <h3 className={classnames('su-type-2 su-mb-03em', levers.bodyHeading, classes.bodyHeading)}>
+              {props.heading}
+            </h3>
+          )}
+
+          <div className={classnames('su-text-normal', levers.body, classes.body)}>
+            {children}
+          </div>
+
+          {props.footer && (
+            <div className={classnames('su-rs-mt-0', levers.footerWrapper, classes.footerWrapper)}>
+              {props.footer}
+            </div>
+          )}
         </div>
       </div>
     </div>
   )
 }
 
-/**
- * Prop Types Information.
- */
+// Prop Types.
+// -----------------------------------------------------------------------------
 
 Alert.propTypes = {
-  // An array of classes to merge with the classes array.
-  classes: PropTypes.object,
-  // The primary content
-  children: PropTypes.node,
-  // FA Icon name.
-  icon: PropTypes.element,
-  // One of the default variant sets.
-  variant: PropTypes.oneOf(Object.keys(variants)),
-  // Show the dismiss button or not.
-  dismiss: PropTypes.bool,
-  // Add a label.
-  label: PropTypes.string,
-  // Add a heading.
-  heading: PropTypes.string,
-  // Add content to the footer region.
-  footer: PropTypes.node
+  // Nodes and content.
+  children: propTypes.oneOfType([
+    propTypes.node,
+    propTypes.element,
+    propTypes.string
+  ]),
+  dismissBtn: propTypes.element,
+  icon: propTypes.element,
+  label: propTypes.string,
+  heading: propTypes.string,
+  footer: propTypes.node,
+
+  // State and Levers.
+  type: propTypes.oneOf(alertTypes),
+  isLargeIcon: propTypes.bool,
+  isLabelTop: propTypes.bool,
+  isIconTop: propTypes.bool,
+  hasDismiss: propTypes.bool,
+  hasIcon: propTypes.bool,
+  hasLabel: propTypes.bool,
+
+  // The CSS Classname property
+  classes: propTypes.shape(
+    {
+      wrapper: propTypes.oneOfType([
+        propTypes.string,
+        propTypes.object,
+        propTypes.array
+      ]),
+      container: propTypes.oneOfType([
+        propTypes.string,
+        propTypes.object,
+        propTypes.array
+      ]),
+      dismissWrapper: propTypes.oneOfType([
+        propTypes.string,
+        propTypes.object,
+        propTypes.array
+      ]),
```

Comment:
It's interesting you did it this way - nice that it allows for some flexibility here.